bcipy/helpers/copy_phrase_wrapper.py

```diff
@@ -97,6 +97,13 @@ def __init__(self,
                  stim_length: int = 10,
                  stim_order: StimuliOrder = StimuliOrder.RANDOM):
 
+        if not lmodel:
```

Comment:
I think we can safely assert this is defined upstream and avoid erroring on image stimuli here? We could then update the type signature to `LanguageModel` and require it as input. What do you think?